lib/team.rb

```diff
@@ -2,15 +2,15 @@ class Team
 
   attr_reader :team_id,
               :franchise_id,
-              :name,
+              :team_name,
               :abbreviation,
               :stadium,
               :link
 
   def initialize(data)
     @team_id = data[:team_id].to_i
     @franchise_id = data[:franchiseid].to_i
-    @name = data[:teamname]
+    @team_name = data[:teamname]
```

Comment:
changed this because one of my tests called for `team_name` and this was an easy fix. Could effed up other people. My bad